infinite/package.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "swr-infinite",
+  "version": "0.0.1",
+  "main": "./dist/index.js",
+  "module": "./dist/index.esm.js",
+  "types": "./dist/infinite",
+  "peerDependencies": {
+    "swr": "*",
+    "react": "*",
+    "dequal/lite": "*"
```

```suggestion
    "dequal": "*"
```

bunchee should be able to resolve the sub folder imports